### ISSUE TITLE
how many epochs when you training?

### ISSUE THREAD

**Issue by cooper87** [rsanchezgarc/deepEMhancer#18]
hi, I have train it for learning, but after I train 100 epochs, the loss just decrease a bit.  how many  epochs when you training?

**Comment by rsanchezgarc** [comment]
Hi @cooper87 ,

We did not train for a fixed number of epochs. Instead, we trained it until the loss stagnated after decreasing the learning rate twice. In practical terms, with our training set of > 70K cubes, the loss function required ~30 epochs to get small enough and another 50 or 60 epochs to get close to the best value.

**Comment by cooper87** [comment]
thanks very much
